spec/graphql/mutations/users/user_metrics_spec.rb

```diff
@@ -44,13 +44,12 @@ def bad_query
           GQL
         end
 
-        xit 'returns user metrics from python service' do
+        it 'returns user metrics from python service' do
           post '/graphql', params: { query: query }
 
           data = parse_json[:data]
 
           create_list(:story, 5, user_id: User.first.id)
-
         end
```

Comment:
I can't remember what this is testing for explicitly, I think we're ok to merge it in but just making a mental note to write out the rest of it :)